src/phoenix/core/evals.py

```diff
@@ -171,3 +173,58 @@ def get_document_evaluation_scores(
                 if result.HasField("score") and document_position < num_documents:
                     scores[document_position] = result.score.value
         return scores
+
+    def export_evaluations(self) -> List[Evaluations]:
+        evaluations: List[Evaluations] = []
+        evaluations.extend(self._export_span_evaluations())
+        evaluations.extend(self._export_document_evaluations())
+        return evaluations
+
+    def _export_span_evaluations(self) -> List[SpanEvaluations]:
+        span_evaluations = []
+        for eval_name, span_evaluations_by_id in self._span_evaluations_by_name.items():
+            span_ids = []
+            rows = []
+            for span_id, pb_eval in span_evaluations_by_id.items():
+                result = pb_eval.result
+                row = {}
+                if result.HasField("score"):
+                    row["score"] = result.score.value
+                if result.HasField("label"):
+                    row["label"] = result.label.value
+                if result.HasField("explanation"):
+                    row["explanation"] = result.explanation.value
+                span_ids.append(span_id)
+                rows.append(row)
+            dataframe = DataFrame(rows, index=Index(span_ids, name="context.span_id"))
+            span_evaluations.append(SpanEvaluations(eval_name, dataframe))
+        return span_evaluations
+
+    def _export_document_evaluations(self) -> List[DocumentEvaluations]:
+        evaluations = []
+        for eval_name, document_evaluations_by_id in self._document_evaluations_by_name.items():
+            span_ids = []
+            document_positions = []
+            rows = []
+            for span_id, document_evaluations_by_position in document_evaluations_by_id.items():
+                for document_position, pb_eval in document_evaluations_by_position.items():
```

Comment:
nit. let's apply `sorted` to `document_position`
```suggestion
                with self._lock:
                    pb_evals = sorted(document_evaluations_by_position.items())
                for document_position, pb_eval in pb_evals:
```

Reply:
Good call

Reply:
addressed in [8be7d5e](https://github.com/Arize-ai/phoenix/pull/2094/commits/8be7d5e9c55527ffa3a1c2e9b6ebf3345209dc1e)